### DISCUSSION TITLE
Mac and Jack

### DISCUSSION THREAD

**Discussion by syneme** [jackaudio/jack2#915] [General]
What's the latest for Macos and Jack? Sorry, I haven't a new mac and can't test. can someone help? I've heard it installs, but I don't know if it's showing up as audio device for normal audio apps like Max/msp, Logic or Ableton. Anyone have a minute? thanks, Ken

**Comment by falkTX** [comment]
jack2 works on macos, the official builds contain qjackctl in order to provide a GUI for start jack as sound server.

there is no way to get normal application audio into jack, they must explicitely support jack in order to be usable within it.
jack does not create any virtual sound interface for other apps to send audio to
